--- dnsmasq.SlackBuild.orig	2019-06-02 06:27:31.409060328 -0400
+++ dnsmasq.SlackBuild	2019-06-02 07:13:58.509126568 -0400
@@ -24,7 +24,7 @@
 VERSION=${VERSION:-$(echo dnsmasq-*.tar.xz | rev | cut -f 3- -d . | cut -f 1 -d - | rev)}
 BUILD=${BUILD:-1_slack14.2}
 
-NUMJOBS=${NUMJOBS:-" -j7 "}
+NUMJOBS=${NUMJOBS:-" -j2 "}
 
 # Automatically determine the architecture we're building on:
 if [ -z "$ARCH" ]; then
@@ -55,7 +55,7 @@
 
 zcat $CWD/dnsmasq.leasedir.diff.gz | patch -p1 --verbose --backup --suffix=.orig || exit 1
 
-make $NUMJOBS all-i18n PREFIX=/usr MANDIR=/usr/man || exit 1
+make $NUMJOBS all-i18n COPTS="-DHAVE_DBUS" PREFIX=/usr MANDIR=/usr/man || exit 1
 make install-i18n PREFIX=/usr DESTDIR=$PKG MANDIR=/usr/man || exit 1
 chmod 0755 $PKG/usr/sbin/dnsmasq
 
